unified-runtime/test/conformance/lit.cfg.py

```diff
@@ -7,5 +7,50 @@
 
 """
 
-# Disable lit testing (for now)
-config.suffixes = []
+import lit.formats
+import subprocess
+from os import path
+
+config.name = "Unified Runtime Conformance"
+config.test_format = lit.formats.GoogleTest("", "-test")
+config.test_source_root = config.binary_dir
+config.test_exec_root = config.binary_dir
+
+config.environment["UR_LOADER_USE_LEVEL_ZERO_V2"] = "1" if config.using_l0_v2 else "0"
+
+if "UR_CTS_ALSO_RUN_KNOWN_FAILURES" in os.environ:
+    config.environment["UR_CTS_ALSO_RUN_KNOWN_FAILURES"] = os.environ[
+        "UR_CTS_ALSO_RUN_KNOWN_FAILURES"
+    ]
+
+if "selector" in lit_config.params:
+    # Specified via passing `-Dselector="backend:*"` to lit
+    selector = lit_config.params["selector"]
+elif os.environ.get("ONEAPI_DEVICE_SELECTOR", ""):
+    lit_config.warning(
+        "ONEAPI_DEVICE_SELECTOR set in environment, using that for UR conformance tests"
+    )
+    selector = os.environ["ONEAPI_DEVICE_SELECTOR"]
+elif config.default_selector:
+    selector = config.default_selector
+else:
+    selector = ";".join([f"{a}:*" for a in config.adapters_built if a != "mock"])
+config.environment["ONEAPI_DEVICE_SELECTOR"] = selector
+
+if selector == "":
+    lit_config.warning(
+        "No device selector is set (are any adapters enabled?), conformance tests may not run as expected"
+    )
+
+lit_config.note(
+    f"Running Unified Runtime conformance tests with ONEAPI_DEVICE_SELECTOR='{selector}':"
+)
+urinfo = subprocess.check_output(
+    path.join(config.runtime_dir, "urinfo"),
+    text=True,
+    env=config.environment,
+    stderr=subprocess.PIPE,
+)
+for l in urinfo.split("\n"):
```

Comment:
`s/split("\n")/splitlines/`

Reply:
Also does this really need to be split? `lit.config.note` will prefix every call with line number program names file and kind. That may or may not be what we want

Reply:
Having the prefix is intentional, otherwise it looks like:

```
lit: /home/ross/dpcpp/unified-runtime/test/conformance/lit.cfg.py:54: note:   [hip:gpu] AMD HIP BACKEND, Radeon RX 7900 GRE gfx1100 [HIP 60342.13]
[level_zero:gpu] Intel(R) oneAPI Unified Runtime over Level-Zero V2, Intel(R) UHD Graphics 770 12.2.0 [1.6.32224.500000]
```

Which doesn't look that great.